rllib/algorithms/ppo/ppo_learner.py

```diff
@@ -73,3 +79,29 @@ def additional_update_per_module(
         results.update({LEARNER_RESULTS_CURR_ENTROPY_COEFF_KEY: new_entropy_coeff})
 
         return results
+
```

Comment:
Why do we need this for this PR? This will conflict with the clean ups in the pipeline for learners in this PR https://github.com/ray-project/ray/pull/35573